sdk/search/test.yml

```diff
@@ -0,0 +1,14 @@
+trigger: none
+
+jobs:
+    - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
+      parameters:
+          ServiceDirectory: search
+          DisableAzureResourceCreation: true
+          Timeout: 120
+          EnvVars:
+              AZURE_TEST_MODE: LIVE
+              AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
+              AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
+              AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
+              AZURE_SUBSCRIPTION_ID: $(aad-azure-sdk-test-subscription-id)
```

Comment:
`yml` files are space sensitive, this is using the wrong character offset. This will either lead to a parsing exception or the configuration being ignored.